init lexical demo

samuelmartineau opened the pull request.

samuelmartineau committed:
init lexical demo

Will956 commented on lexical/src/editor/Actions.tsx:
```diff
@@ -0,0 +1,25 @@
+  const { nodeKey } = useContext(RichLegislationContext);
+
+  return (
+    <div className="">
```
```suggestion
    <div>
```

Will956 commented on lexical/src/editor/Editor.tsx:
```diff
@@ -0,0 +1,54 @@
+      }}
+    >
+      <div className="mx-auto bg-white rounded-xl shadow-md overflow-hidden ">
+        <div className="m-8 flex space-y-8 flex-col	">
```
```suggestion
        <div className="m-8 flex space-y-8 flex-col">
```

Will956 commented on lexical/src/editor/Editor.tsx:
```diff
@@ -0,0 +1,54 @@
+        onError: console.error,
+      }}
+    >
+      <div className="mx-auto bg-white rounded-xl shadow-md overflow-hidden ">
```
```suggestion
      <div className="mx-auto bg-white rounded-xl shadow-md overflow-hidden">
```

Will956 commented on lexical/src/editor/Editor.tsx:
```diff
@@ -0,0 +1,54 @@
+                  <ContentEditable className="p-2 min-h-[150px] border-solid border-2 border-sky-500" />
+                }
+                placeholder={
+                  <div className="absolute top-2 left-2 pointer-events-none	">
```
```suggestion
                  <div className="absolute top-2 left-2 pointer-events-none">
```

Will956 commented on lexical/src/index.tsx:
```diff
@@ -0,0 +1,19 @@
+  </React.StrictMode>
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Useless pour la demo ? Surement un truc qui vient automatiquement avec le setup mais ça permettrai d'alleger le code 👍

samuelmartineau committed:
fix typos

Co-authored-by: William Duplenne <will.dupl95@gmail.com>

Update lexical/src/editor/Editor.tsx

Co-authored-by: William Duplenne <will.dupl95@gmail.com>

Update lexical/src/editor/Editor.tsx

Co-authored-by: William Duplenne <will.dupl95@gmail.com>

Update lexical/src/editor/Actions.tsx

Co-authored-by: William Duplenne <will.dupl95@gmail.com>

samuelmartineau committed:
simplify demo boilerplate

samuelmartineau committed:
bump lexical

samuelmartineau committed:
add doc

samuelmartineau committed:
publish build folder
